helm/argocd-test/action.yml

```
@@ -0,0 +1,42 @@
+name: "Test that an helm chart renders"
+description: "Test that an helm chart renders"
+
+inputs:
+  chart-dir:
+    description: "Specify the chart directory."
+    required: true
+    default: "argocd/base/"
+  helm-option:
```

Make it plural to be consistent
```suggestion
  helm-options:
```